include/mbedtls/asn1.h

```diff
@@ -198,7 +198,7 @@ typedef struct mbedtls_asn1_named_data {
 mbedtls_asn1_named_data;
 
 #if defined(MBEDTLS_ASN1_PARSE_C) || defined(MBEDTLS_X509_CREATE_C) || \
-    defined(MBEDTLS_PSA_UTIL_HAVE_ECDSA)
+    defined(MBEDTLS_PSA_UTIL_HAVE_ECDSA) || defined(MBEDTLS_PEM_PARSE_C)
```

Comment:
Note: PEM doesn't always directly require ASN1, only when it supports RFC1421 encryption, so technically the condition here should be . But PEM is useful pretty much only as a container for DER, so a configuration with PEM but not ASN1 parsing wouldn't be useful. So depending only on `MBEDTLS_PEM_PARSE_C` and not `… && RFC1421` is fine.